src/uk/ac/ebi/vfb/neo4j/KB_tools.py

```diff
@@ -999,12 +999,12 @@ def add_anatomy_image_set(self,
         if dbxrefs:
             for db, acc in dbxrefs.items():
                 self.ew.add_annotation_axiom(s=anat_id['short_form'],
-                                             r='hasDbXref',
+                                             r='database_cross_reference',
```

Comment:
As match_on is set to 'short_form' (see arg below), this must be the short_form of 'http://www.geneontology.org/formats/oboInOwl#hasDbXref' in the database, which = 'hasDbXref'.  So this won't work.